### ISSUE TITLE
steps contentful - resources (and other lists)

### ISSUE THREAD

**Issue by claudioccm** [GovLab/data-collaboratives#57]
since we are importing this content as markdown, we need to consistent with how this data is inserted. 

Lists NEED to be inserted as bullets or numbers. They will be visually a list (doesnt matter is they are `ul`, or `ol`). But they CANNOT be multiple `p`. 

This is happening in STEP 1: Resources.

**Comment by chrswng** [comment]
yeah agreed, that's the biggest drawback to providing markdown editing for non-dev team. fortunately we can define how it's done, and then tell them to do it the same way every time. i had used `<p>` when i input originally just bc i was trying to get some data in, but yeah, will go back and change to list. 

we should document any other standard elements we want them to use in the future and perhaps we can create readme files for contentful input.

**Comment by claudioccm** [comment]
ok, sounds good.
